### ISSUE TITLE
PHP 8 Support

### ISSUE THREAD

**Issue by shyim** [shopwareLabs/psh#102]
```
Fatal error: Uncaught TypeError: abs(): Argument #1 ($number) must be of type int|float, string given in phar:///builds/shopware/5/product/shopware/psh.phar/vendor/symfony/yaml/Parser.php:554
Stack trace:
#0 phar:///builds/shopware/5/product/shopware/psh.phar/vendor/symfony/yaml/Parser.php(554): abs('')
#1 phar:///builds/shopware/5/product/shopware/psh.phar/vendor/symfony/yaml/Parser.php(254): Symfony\Component\Yaml\Parser->parseValue('|', 0, 'mapping')
#2 phar:///builds/shopware/5/product/shopware/psh.phar/src/Config/YamlConfigFileLoader.php(102): Symfony\Component\Yaml\Parser->parse('header: |\n  SHO...')
#3 phar:///builds/shopware/5/product/shopware/psh.phar/src/Config/YamlConfigFileLoader.php(53): Shopware\Psh\Config\YamlConfigFileLoader->parseFileContents('header: |\n  SHO...')
#4 phar:///builds/shopware/5/product/shopware/psh.phar/src/Application/ApplicationFactory.php(41): Shopware\Psh\Config\YamlConfigFileLoader->load('/builds/shopwar...', Array)
#5 phar:///builds/shopware/5/product/shopware/psh.phar/src/Application/Application.php(57): Shopware\Psh\Application\ApplicationFactory->createConfig('/builds/shopwar...', Array)
#6 phar:///builds/shopware/5/product/shopware/psh.phar/psh(12): Shopware\Psh\Application\Application->run(Array)
#7 /builds/shopware/5/product/shopware/psh.phar(10): require('phar:///builds/...')
#8 {main}
  thrown in phar:///builds/shopware/5/product/shopware/psh.phar/vendor/symfony/yaml/Parser.php on line 554
```

**Comment by JanPietrzyk** [comment]
This issue is resolved by the update pr #106

**Comment by JanPietrzyk** [comment]
should be fixed, we will take a closer look once php8 is released
